src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+function createMessage(text, className = 'message') {
+  const div = document.createElement('div');
+
+  div.classList = className;
```

Comment:
Use `div.className = className;` instead of `div.classList = className;` to correctly assign a class to the `div` element. `classList` is used for adding, removing, or toggling classes, not for direct assignment.